draft-ietf-httpbis-priority.md

```diff
@@ -185,136 +185,71 @@ In both cases, the set of priority parameters is encoded as a Structured Headers
 Dictionary ({{!STRUCTURED-HEADERS}}).
 
 This document defines the urgency(`u`) and incremental(`i`) parameters. When
-used, these parameters MUST be accompanied by values. When any of the defined
-parameters are omitted, or if the Priority header field is not used, their
-default values SHOULD be applied.
+receiving an HTTP request that does not carry these priority parameters, a
+server SHOULD act as if the default values were specified.
```

Comment:
I see that the HTTP response case is mentioned later but I think it might help to explicitly mention that it here e.g. a response with the parameters SHOULD NOT act be interpreted as if default were specified.

Reply:
Done in 146fd30. PTAL.